src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10Driver.notSupported.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A GPIO driver for Windows 10 IoT.
+    /// </summary>
+    public class Windows10Driver : GpioDriver
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Windows10Driver"/> class.
+        /// </summary>
+        public Windows10Driver()
+        {
+            // If we land in this method it means the console application is running on Windows and targetting net5.0 (without specifying Windows platform)
+            // In order to call WinRT code in net5.0 it is required for the application to target the specific platform
+            // so we throw the bellow exception with a detailed message in order to instruct the consumer on how to move forward.
+            throw new PlatformNotSupportedException("In order to use Windows10Driver on Windows with .NET 5.0 it is required for your application to target net5.0-windows10.0.17763.0. Please add that to your target frameworks in your project file.");
```

Comment:
Can you factor this to a common resource string with parameter (`{0}`) for type name to avoid duplication?  Perhaps you might also want to parameterize that windows version?  Not sure how you plan to keep that in sync with TargetFramework.

Reply:
The repo doesn't currently use ResourceManager and it is in our plans to add that support, but ideally, when added I would rather do it to the whole library and not just this string. I can centralize the format of this string for now as some sort of constant format, and then for vNext work on moving everything into resources instead, does that sound good?

Reply:
Even though you aren't using ResX here you can still de-dup the string to a single helper method.

Reply:
yeah, that's what I meant with centralizing the format. I'll do that then.

Reply:
Yep, sorry I had replied here before seeing your reply.  GH didn't refresh for some reason.